lib/bpk-component-emoji/src/BpkEmoji.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React from 'react';
+import { Text, ViewPropTypes, StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  emoji: {
+    fontFamily: 'System',
```

Comment:
You might need to make this `sans-serif` if `Platform.OS === 'android'` to avoid the Danger error about iOS tokens in snapshots. `System` isn't a font on Android, so the reason it's probably working right now is because Android is quietly falling back to `sans-serif` (I assume)